jbpm-case-mgmt/jbpm-case-mgmt-api/src/main/java/org/jbpm/casemgmt/api/CaseRuntimeDataService.java

```diff
@@ -184,7 +184,14 @@
      *
      */
     Collection<CaseInstance> getCaseInstances(QueryContext queryContext);
-   
+
+    /**
+     * Returns all first level children cases given an parent id case.
+     * @param parentCaseId parent case
+     * @param statuses 
+     * @param queryContext
```

Comment:
missing javadoc coments, as in other methods (clarify that statuses are for subcases):
 _@param statuses list of statuses that subcases should be in to match
   @param queryContext control parameters for the result e.g. sorting, paging_